SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -102,14 +117,38 @@ public final class SwiftGenStoryboardEnumBuilder {
         if enumName != nil {
             text += "\(t)}\n"
         }
-        text += "}\n\n"
+      
+        /// Segues
+        if (storyboardsSegues.count > 0) {
+          text += "\n"
+          
+          let enumName = "Segue"
```

Comment:
That should probably be configurable in the builder as a parameter for the `build()` function (replace that single `enumName` parameter with `sceneEnumName`+`segueEnumName`)?